### ISSUE TITLE
Transfer of Ownership

### ISSUE THREAD

**Issue by RandomDSdevel** [BjarneStroustrup/Programming-_Principles_and_Practice_Using_Cpp#1]
@BjarneStroustrup:  

Per (and quoted directly from) [the 'GitHub Repository for 'Programming: Principles and Practice Using C++' Support Code?' thread's original post](https://groups.google.com/a/isocpp.org/forum/#!topicsearchin/std-discussion/author$3Ame/std-discussion/44wUad04BE8) on [the ISO C++ std-discussion reflector](https://groups.google.com/a/isocpp.org/forum/#!forum/std-discussion) from when I tried to reach you concerning the subject a while back:  

> Dear Mr. Stroustrup, 
>
> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;I think it would very convenient and useful if you were to put the support code that you have published on your web page for _Programming:  Principles and Practice Using C++_.  Doing so would allow users like myself to be more proactive in suggesting and contributing improvements and code errata.  For example, end users have previously contributed build support for additional compilers.  I, myself, have created [a GitHub repository containing the support code for the second edition of the book](https://github.com/RandomDSdevel/Programming-_Principles_and_Practice_Using_Cpp) and, even though I haven’t yet gotten around to doing it, plan to add better support for Xcode, either directly via an Xcode template or indirectly through a CMake build profile.  It would be nice if there were a central version of such a repository for me to submit my changes to with a pull request.  A central location for the support files might also allow me to port my efforts back to the code for the book’s first edition as well.  Could you, perhaps, look into setting this up for your readers?  
>
> Sincerely, 
> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bryce Glover
> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RandomDSdevel@gmail.com

P. S.:  _Yes_, I _know_ I should have gotten my contributions ready to merge by now, but I got distracted/derailed…oh, well; such is life (mine especially, it seems these days…_sigh_.)

**Comment by RandomDSdevel** [comment]
@BjarneStroustrup:  I just began transferring this repository's ownership to you via its GitHub settings.  If you wish for me to abort this action, please let me know here.  Once this ownership transfer is complete, I will create my own fork of your new repository.  (If you just want me to wait until you create your _own_ repository containing similar files to this one, then I'll delete this repo and fork the new one.)

**Comment by BjarneStroustrup** [comment]
OK. Thanks.

**Comment by RandomDSdevel** [comment]
@BjarneStroustrup:  You're welcome!  Before I go, though, I should _probably_ note that you _might_ want to clean up this repository's commit history a bit (to perhaps squash commits and fix authorship metadata notations to your liking,) as my initial efforts trying to figure out why I seemed to be having trouble committing some files here left behind a bit of a mess (turns out it was due to journaled HFS+'s default semantics of being case-preserving but case-insensitive, which would of course be a problem to a Mac user like myself, but I've been doing what little development work I've managed since I figured that out off of an 'HFS+, Case-Sensitive, Journaled' disk image for quite some time now, so everything's ship-shape down here on my end now.)  

P. S.:  Since I'll be contributing via pull requests from a fork from now on, I don't need to be added to this repository as a co-maintainer, so you can undo that.  (In addition, it _appears_ GitHub seems to have conflated our two user account identities somehow, as this source repository is still listed as 'mine' on GitHub's home page under 'Your repositories' alongside my fork of it when I visit that page, strange as that sounds…)

**Comment by RandomDSdevel** [comment]
@BjarneStroustrup:  Should I split my [last comment](https://github.com/BjarneStroustrup/Programming-_Principles_and_Practice_Using_Cpp/issues/1#issuecomment-294308529)'s postscript into a new issue, or are you fine with the request I made in it and just haven't gotten around to fulfilling it yet?

**Comment by RandomDSdevel** [comment]
@BjarneStroustrup:  I went ahead and forked the discussion present in this issue to further elaborate on my misgivings as to how you resolved it into [Issue #3:  'Resolving Code Origin, Attribution, and Access Permissions'](https://github.com/BjarneStroustrup/Programming-_Principles_and_Practice_Using_Cpp/issues/3).  It's quite formal (especially coming from _me_,) but I believe it suits the purpose of getting the points it makes across reasonably well regardless of this fact.  Please take a look at and respond there to it.
